src/lib.rs

```diff
@@ -344,6 +348,25 @@ impl PTouch {
         Ok(s)
     }
 
+    // This function is gonna be called if the --no-status-fetch flag is enabled.
+    // It returns a default status, which is assumed to be correct to then print.
+    pub fn status_default(&mut self, media: &Media) -> Result<Status, Error> {
```

Comment:
this doesn't (can't?) use the printer context, so could probably be a method on `Status`? maybe `new(media: &Media) -> Self`?

Reply:
That's right, I'm moving it !